### ISSUE TITLE
Feature Request - allow bulk uploading of file observables

### ISSUE THREAD

**Issue by BrevilleBro** [CERT-BDF/TheHive#327]
# Allow bulk uploading of file observables

### Request Type
Feature Request

### Possible Solutions
Pretty self-explanatory, allow bulk uploading of file observables :)

**Comment by ralliauto** [comment]
In addition, enabling uploading bulk files to task and also to case would be a good feature.

**Comment by saadkadhi** [comment]
Yes, that would make sense. We'll try to make it happen for 3.1.0 (end of April / beg of May 18).
